pkg/storaged/content-views.jsx

```diff
@@ -396,11 +416,21 @@ function append_row(client, rows, level, key, name, desc, tabs, job_object) {
         { name: last_column, tight: true },
     ];
 
+    function menuitem(action) {
+        return <StorageMenuItem key={action.title} onClick={action.func}>{action.title}</StorageMenuItem>;
+    }
+
+    var menu = null;
+    if (tabs.menu_actions && tabs.menu_actions.length > 0)
+        menu = <StorageBarMenu id={"menu-" + name}>{tabs.menu_actions.map(menuitem)}</StorageBarMenu>;
+
+    var actions = <>{tabs.actions}{menu}</>;
+
     rows.push(
         <ListingRow key={key}
                     columns={cols}
                     tabRenderers={tabs.renderers}
-                    listingActions={tabs.actions} />
+        listingActions={actions} />
```

Comment:
Just in case you push again, please re-fix this whitespace damage.

Reply:
Upps, yes.